index.html

```diff
@@ -13612,62 +13612,64 @@ <h2>Interface Mixin <dfn>ARIAMixin</dfn></h2>
 
 	<section id="accessibilityroleandproperties-correspondence" class="normative" data-dfn-for="ARIAMixin" data-link-for="ARIAMixin">
 		<h2>ARIA Attribute Correspondence</h2>
-		<p>The following table provides a correspondence between IDL attribute names and content attribute names, for use by <code>ARIAMixin</code>.</p>
+		<p>The following table provides a correspondence between IDL attribute names, content attribute names, and their value types, for use by <code>ARIAMixin</code>.</p>
```

Comment:
Since ARIAMixin doesn't use the value type, perhaps this should be clarified? E.g., "It also lists correspondence to value type for informative purposes."